### PR TITLE
Pruning dead code

### PR DESCRIPTION
**Pull request by devdoshi**
## Why
Describe the motivation for the changes.

## What
What's changing? Anything of note to call out?

## Validation
* [ ] CI passes
* [ ] Verified in staging
* etc...

### CODE REVIEW

File: .env.latest [hunk 1]
```diff
@@ -1,11 +1,7 @@
 OPTIC_ASSEMBLED_SPEC_EVENTS=false
 OPTIC_SPLIT_SPEC_EVENTS=false
 OPTIC_RUST_DIFF_ENGINE=true
-OPTIC_RUST_INITIAL_BODY_LEARNER=false
-OPTIC_RUST_SHAPE_DIFF_AFFORDANCES_LEARNER=false
-REACT_APP_LEARN_API_MODE=true
 REACT_APP_ENABLE_ANALYTICS=yes
```
Comment (devdoshi) [inline .env.latest:4]: @acunniffe make sure you're respecting this value

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -243,54 +245,13 @@ jobs:
...
-          cp $FLAGS_FILE .env
-          cp $FLAGS_FILE workspaces/agent-cli/.env
-          cp $FLAGS_FILE workspaces/ci-cli/.env
-          cp $FLAGS_FILE workspaces/local-cli/.env
-          cp $FLAGS_FILE workspaces/cli-server/.env
-          cp $FLAGS_FILE workspaces/cli-scripts/.env
-          cp $FLAGS_FILE workspaces/diff-engine/.env
-          cp $FLAGS_FILE workspaces/ui/.env.production.local
-        fi
-
```
Comment (JaapRood) [inline .github/workflows/release.yml:257]: This lives in the task file now, right?

**Review by JaapRood — APPROVED** [review]
Looks all good from my perspective, with one question that's more about me not sure where that stuff lives now (I _have_ seen it around _somewhere_).

File: .github/workflows/release.yml (hunk 2 above)
Comment (devdoshi) [reply .github/workflows/release.yml:257]: correct!

**Pull request merged**